live-rates/test/adapter.test.ts

```diff
@@ -0,0 +1,81 @@
+import { assert } from 'chai'
+import { Requester } from '@chainlink/external-adapter'
+import { assertSuccess, assertError } from '@chainlink/adapter-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { execute } from '../src/adapter'
+
+describe('execute', () => {
+  const jobID = '1'
+
+  context('successful calls @integration', () => {
+    const requests = [
+      {
+        name: 'id not supplied',
+        testData: { data: { symbol: 'EURUSD' } },
+      },
+      {
+        name: 'symbol EURUSD',
+        testData: { id: jobID, data: { symbol: 'EURUSD' } },
+      },
+      {
+        name: 'symbol BTCEUR',
+        testData: { id: jobID, data: { symbol: 'BTCEUR' } },
+      },
+      {
+        name: 'from',
+        testData: { id: jobID, data: { from: 'EURUSD' } },
```

Comment:
I would prefer seeing the tests use `to` and `from`: `{from: 'ETH', to: 'USD'}`

Reply:
I implemented the changes as suggested by you! thanks for your feedback @boxhock!